examples/images_contours_and_fields/image_antialiasing.py

```diff
@@ -72,6 +74,73 @@
     ax.set_title(f"interpolation='{interp}'")
 plt.show()
 
+###############################################################################
+# Data antialiasing and RGBA antialiasing
+# ----------------------------------------
+#
+# The examples above all used grayscale.  When colormapping is added there
+# is the complication that downsampling and the antialiasing filters are
+# applied to the data in Matplotlib, before the data is mapped to
+# colors.  So in the following note how the corners fade to white, the middle
+# of the colormap, because the data is being smoothed and the high and low
+# values are averaging out to the middle of the colormap.
+
+f0 = 10
+k = 150
+a = np.sin(np.pi * 2 * (f0 * R + k * R**2 / 2))
+
+fig, axs = plt.subplots(1, 2, figsize=(7, 4), sharex=True, sharey=True,
+                        constrained_layout=True)
+for ax, interp in zip(axs, ['nearest', 'antialiased']):
+    pc = ax.imshow(a, interpolation=interp, cmap='RdBu_r', vmin=-1, vmax=1)
+    ax.set_title(f"'{interp}'", fontsize='small')
+fig.colorbar(pc, ax=axs)
+plt.show()
+
+###############################################################################
+# Sometimes, however, we want the antialiasing to occur in RGBA space.  In
+# this case purple is actually the perceptual mixture of red and blue.
+# Matplotlib doesn't allow this to be directly achieved, but we can pass
+# RGBA data to `~.Axes.imshow`, and then the antialiasing filter will be
+# applied to the RGBA data:
+
+fig, axs = plt.subplots(1, 2, figsize=(7, 4), sharex=True, sharey=True,
+                        constrained_layout=True)
+norm = mcolors.Normalize(vmin=-1, vmax=1)
+cmap = cm.RdBu_r
+a_rgba = cmap(norm(a))
+for ax, interp in zip(axs, ['nearest', 'antialiased']):
+    pc = ax.imshow(a_rgba, interpolation=interp)
+    ax.set_title(f"'{interp}'", fontsize='small')
+plt.show()
+
+###############################################################################
+# A concrete example of where antialiasing in data space may not be desirable
+# is given here, where the anti-aliasing returns white pixels in data space,
+# and (imperceptible) purple pixels in RGBA space:
+
+fig, axs = plt.subplots(1, 3, figsize=(5.5, 2), sharex=True, sharey=True,
+                        constrained_layout=True)
+f0 = 10
+k = 100
+a = np.sin(np.pi * 2 * (f0 * R + k * R**2 / 2))
+
+aa = a
+aa[np.sqrt(R) < 0.6] = 1
+aa[np.sqrt(R) < 0.5] = -1
+
+norm = mcolors.Normalize(vmin=-1, vmax=1)
+cmap = cm.RdBu_r
+a_rgba = cmap(norm(aa))
+
+axs[0].imshow(aa, interpolation='nearest', cmap='RdBu_r')
+axs[0].set_title('No antialiasing')
```

Comment:
Interestingly, the titles are clipped at the top:
https://52747-1385122-gh.circle-artifacts.com/0/doc/build/html/gallery/images_contours_and_fields/image_antialiasing.html#sphx-glr-gallery-images-contours-and-fields-image-antialiasing-py
![grafik](https://user-images.githubusercontent.com/2836374/106530003-c0d1d000-64eb-11eb-8a3a-d8fcc09a0431.png)

Reply:
Yeah, CL sometimes has burbles like that if it is shrinking to fit rather than growing to fit.  I just made the figure a bit bigger